pkg/ovs/ovn-nb-logical_switch_port.go

```diff
@@ -25,6 +25,14 @@ func (c *OVNNbClient) CreateLogicalSwitchPort(lsName, lspName, ip, mac, podName,
 
 	// ignore
 	if exist {
+		if portSecurity && len(securityGroups) != 0 {
+			lsp, err := c.GetLogicalSwitchPort(lspName, true)
```

Comment:
please log the err  if err != nil, and then set the SG

Reply:
updated.